### PR TITLE
Add User interface design 

### PR DESCRIPTION
**Pull request by adejam**
The PR does the following:

- Add navbar to design
- Implement styling to the components.
- [live link](https://deploy-preview-5--booklyb.netlify.app/)

### CODE REVIEW

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Nice job with the design. 

Please move the filter to the category section in the nav bar.

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        </a>
+        <nav>
+          <a href="#home" className="navItem books tt_u">
+            Books
+          </a>
+          <CategoryFilter filterHandler={handleFilterChange} />
+        </nav>
+        <nav className="mlAuto">
+          <a href="/avatar" className="avatar  navItem">
+            <img src="#" width="15" height="15" alt="A" />
```
Comment (bolah2009) [inline src/components/Navbar.js]: Why not make this a blue circle with div and "A" in it as white text, makes more sense that a broken image, or alternatively add an image.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <header className="bgWhite">
+      <nav className="navbar" id="myTopnav">
+        <a className="navbarBrand azure" href="/">
+          Book Store
+        </a>
+        <nav>
+          <a href="#home" className="navItem books tt_u">
+            Books
+          </a>
+          <CategoryFilter filterHandler={handleFilterChange} />
```
Comment (bolah2009) [inline src/components/Navbar.js:21]: Could you style this better and make the mouse recognise it as a clickable element? Currently it' too hidden and may not be obvious that it' changeable. Maybe add a `cursor: pointer;` CSS property, that way, hovering it makes it more obvious it's clickable. I would also recommend giving it a border when the mouse hovers it.

**Review by bolah2009 — APPROVED** [review]
LTGM! 🚀

File: src/components/Navbar.js (hunk 1 above)
Comment (adejam) [reply src/components/Navbar.js]: I'll go with adding a blue div with "A".

File: src/components/Navbar.js (hunk 2 above)
Comment (adejam) [reply src/components/Navbar.js:21]: I'll add the effect

**Pull request merged**